src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    static final int MAX_COUN_FIGURE = 4;
+    static final int MAX_POS_NUMBER = 100;
+
+    public Figure getRandomFigure() {
+        Figure figure = null;
+        Random random = new Random();
+        switch (new Random().nextInt(MAX_COUN_FIGURE)) {
+            case 0:
+                figure = new Circle(random.nextInt(MAX_POS_NUMBER));
+                break;
+            case 1:
+                figure = new IsoscelesTrapezoid(random.nextInt(MAX_POS_NUMBER),
+                        random.nextInt(MAX_POS_NUMBER), random.nextInt(MAX_POS_NUMBER));
+                break;
+            case 2:
+                figure = new Rectangle(random.nextInt(MAX_POS_NUMBER),
+                        random.nextInt(MAX_POS_NUMBER));
+                break;
+            case 3:
+                figure = new RightTriangle(random.nextInt(MAX_POS_NUMBER),
+                        random.nextInt(MAX_POS_NUMBER));
+                break;
+            case 4:
+                figure = new Square(random.nextInt(MAX_POS_NUMBER));
```

Comment:
check documentation
`nextInt(int bound)
Returns a pseudorandom, uniformly distributed int value between 0 (inclusive) and the specified value (exclusive), drawn from this random number generator's sequence.
nextInt(int bound)
Returns a pseudorandom, uniformly distributed int value between 0 (inclusive) and the specified value (exclusive), drawn from this random number generator's sequence.` from https://docs.oracle.com/javase/8/docs/api/java/util/Random.html